### PR TITLE
Use 20.07-RC1, add skip_qc to MD

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
# nf-core/sarek pull request

Many thanks for contributing to nf-core/sarek!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/sarek branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/sarek)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -258,8 +258,9 @@ include { BWAMEM2_MEM }           from './modules/local/bwamem2_mem.nf'
 include { GET_SOFTWARE_VERSIONS } from './modules/local/get_software_versions'
 include { OUTPUT_DOCUMENTATION }  from './modules/local/output_documentation'
 include { TRIM_GALORE }           from './modules/local/trim_galore.nf'
-include { MERGE_BAM_MAPPED }      from './modules/local/merge_mapped_bam' addParams(params)
-include { MARK_DUPLICATES }       from './modules/local/mark_duplicates' params(params)
+include { MERGE_BAM_MAPPED }      from './modules/local/merge_mapped_bam' 
+include { MARK_DUPLICATES }       from './modules/local/mark_duplicates' addParams(skip_qc: skip_qc)
```
Comment (MaxUlysse) [inline main.nf:262]: I see, now I understand better

File: modules/local/mark_duplicates.nf [hunk 2]
```diff
@@ -11,7 +11,7 @@ process MARK_DUPLICATES {
      output:
          tuple idPatient, idSample, path("${idSample}.md.bam"), path("${idSample}.md.bam.bai"), emit:  bam_duplicates_marked
          tuple idPatient, idSample, emit: tsv_bam_duplicates_marked
-         path "${idSample}.bam.metrics", emit: duplicates_marked_report //is optional , applies when skip_qc is used(not implemented yet)
+         path "${idSample}.bam.metrics", optional : true, emit: duplicates_marked_report //is optional , applies when skip_qc is used(not implemented yet)
```
Comment (MaxUlysse) [inline modules/local/mark_duplicates.nf:14]: Actually we will need to rename this option.

you can't actually skip markduplicates with skip_qc, but you actually don't keep the report.

Comment (FriederikeHanssen) [reply modules/local/mark_duplicates.nf:14]: Ah, I see. So maybe skip_markduplicates_report and make it an extra parameter not included in the skip_qc list?

**Comment by FriederikeHanssen** [comment]
@MaxUlysse I am done adding things to this PR. Sorry, I wanted to add the new nextflow.enable thing and then had to fix some things after that

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         # Nextflow versions: check pipeline minimum and current latest
-        nxf_ver: ['20.04.1', '']
+        nxf_ver: ['20.07.0-RC1', '']
```
Comment (MaxUlysse) [inline .github/workflows/ci.yml:15]: Can you update the README.md too?

Comment (FriederikeHanssen) [reply .github/workflows/ci.yml:15]: done

**Pull request merged**